lavamoat/browserify/flask/policy.json

```diff
@@ -989,6 +988,7 @@
         "@metamask/controller-utils>@spruceid/siwe-parser": true,
         "bn.js": true,
         "browserify>buffer": true,
+        "cockatiel": true,
         "eth-ens-namehash": true,
```

Comment:
This is expected and been added [here](https://github.com/MetaMask/core/pull/5141/files#diff-84d97bcb8e5e8d4e4abc52abeb82b49d1fb29331788b5614c1a1e2ae94bb59cf)